vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java

```diff
@@ -72,7 +72,8 @@
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
-    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
+    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(27, 43,
+              65, 0.72d)
             128, 0.94);
```

Comment:
This was a leftover from my suggestion, as GitHub didn't allow me to extend a suggestion over a deleted line.

```suggestion
```